app/stac_api/migrations/0016_auto_20210902_1731.py

```diff
@@ -0,0 +1,119 @@
+                    ('application/x.tab+zip', 'Zipped Mapinfo-TAB (application/x.tab+zip)'),
+                    (
+                        'application/x.tab-raster+zip',
+                        'Zipped Mapinfo-Raster-TAB (application/x.tab-raster+zip)'
+                    ), ('application/x.csv+zip',
+                        'Zipped CSV (application/x.csv+zip)'), ('text/csv', 'CSV (text/csv)'), (
+                            'application/geopackage+sqlite3',
+                            'Geopackage (application/geopackage+sqlite3)'
+                        ),
+                    (
+                        'application/x.geopackage+zip',
+                        'Zipped Geopackage (application/x.geopackage+zip)'
+                    ), ('application/geo+json', 'GeoJSON (application/geo+json)'),
+                    ('application/x.geojson+zip', 'Zipped GeoJSON (application/x.geojson+zip)'),
+                    (
+                        'application/x.interlis; version=2.3',
+                        'Interlis 2 (application/x.interlis; version=2.3)'
+                    ),
+                    (
+                        'application/x.interlis+zip; version=2.3',
+                        'Zipped XTF (2.3) (application/x.interlis+zip; version=2.3)'
+                    ),
+                    (
+                        'application/x.interlis; version=1',
+                        'Interlis 1 (application/x.interlis; version=1)'
+                    ),
+                    (
+                        'application/x.interlis+zip; version=1',
+                        'Zipped ITF (application/x.interlis+zip; version=1)'
+                    ),
+                    (
+                        'image/tiff; application=geotiff; profile=cloud-optimized',
+                        'Cloud Optimized GeoTIFF (COG) (image/tiff; application=geotiff; profile=cloud-optimized)'
+                    ), ('application/pdf', 'PDF (application/pdf)'),
+                    ('application/x.pdf+zip', 'Zipped PDF (application/x.pdf+zip)'),
+                    ('application/json', 'JSON (application/json)'),
+                    ('application/x.json+zip', 'Zipped JSON (application/x.json+zip)'),
+                    ('application/x-netcdf', 'NetCDF (application/x-netcdf)'),
+                    ('application/x.netcdf+zip', 'Zipped NetCDF (application/x.netcdf+zip)'),
+                    ('application/xml', 'XML (application/xml)'),
+                    ('application/x.xml+zip', 'Zipped XML (application/x.xml+zip)'),
+                    (
+                        'application/vnd.mapbox-vector-tile',
+                        'mbtiles (application/vnd.mapbox-vector-tile)'
+                    ), ('text/plain', 'Text (text/plain)'),
+                    ('text/x.plain+zip', 'Zipped text (text/x.plain+zip)'),
+                    ('application/x.dwg+zip', 'Zipped DWG (application/x.dwg+zip)')
```

Comment:
There was a recent PR (https://github.com/geoadmin/service-stac/pull/312) for adding this new media type. I did not have on my mind, that an according migrations file is needed for that, too. Good to have it here then ;-)